doc/source/faq.rst

```diff
@@ -18,3 +18,22 @@ Below you can find some answers to frequently asked questions:
   If Java is not installed on the remote system the two tasks for detecting EAP
   will not complete successfully and will not display EAP installation if they
   are present.
+
+* What delimiter does rho use for the CSV report files?
+
+
+  Rho generates its report in a comma seperated format using ',' as the
+  delimiter. Most office software can import this, but you must indicate
+  that the delimiter is a comma, as some ".csv" files use tabs or semicolons.
+
+  Importing the file with the incorrect delimiter or a mix of delimiters can
+  cause the report to be displayed incorrectly, with the wrong data mapped to the
+  headings.
+
+* I can't open my report with my office software! (libreoffice, sheets, excel, etc.)
+
+  Note that the file ending of the report file is determined by the user. If
+  you are using your report in a script, it is probably not important what the
+  file ending is, but it may be necessary for your office software to detect
+  the format. So you should pass :code:`--reportfile=nameofreport.csv` with the
```

Comment:
Instead of "so you should pass ...", I would say "be sure the report filename has `.csv` at the end to help your office software detect its format.